01.multithreading/MultiThreading.Task2.Chaining/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿/*
+ * 2.	Write a program, which creates a chain of four Tasks.
+ * First Task – creates an array of 10 random integer.
+ * Second Task – multiplies this array with another random integer.
+ * Third Task – sorts this array by ascending.
+ * Fourth Task – calculates the average value. All this tasks should print the values to console.
+ */
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MultiThreading.Task2.Chaining
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine(".Net Mentoring Program. MultiThreading V1 ");
+            Console.WriteLine("2.	Write a program, which creates a chain of four Tasks.");
+            Console.WriteLine("First Task – creates an array of 10 random integer.");
+            Console.WriteLine("Second Task – multiplies this array with another random integer.");
+            Console.WriteLine("Third Task – sorts this array by ascending.");
+            Console.WriteLine("Fourth Task – calculates the average value. All this tasks should print the values to console");
+            Console.WriteLine();
+            Random r = new Random();
+
+            Task<int> task = Task.Run(() =>
```

Comment:
`var` keyword can be used instead of `Task<int>`

Reply:
done